File: lib/src/core/platform_common_providers.dart
library angular2.src.core.platform_common_providers;

import "package:angular2/src/facade/lang.dart"
    show Type, isBlank, isPresent, assertionsEnabled;
import "package:angular2/src/core/di.dart"
    show provide, Provider, Injector, OpaqueToken;
import "package:angular2/src/core/console.dart" show Console;
import "reflection/reflection.dart" show Reflector, reflector;
import "package:angular2/src/core/testability/testability.dart"
    show TestabilityRegistry;

Reflector _reflector() {
  return reflector;
}

/**
 * A default set of providers which should be included in any Angular platform.
 */
const List<
        dynamic /* Type | Provider | List < dynamic > */ > PLATFORM_COMMON_PROVIDERS =
    const [
  const Provider(Reflector, useFactory: _reflector, deps: const []),
  TestabilityRegistry,
  Console
];

File: es6/prod/src/core/platform_common_providers.js
import { CONST_EXPR } from 'angular2/src/facade/lang';
import { Provider } from 'angular2/src/core/di';
import { Console } from 'angular2/src/core/console';
import { Reflector, reflector } from './reflection/reflection';
import { TestabilityRegistry } from 'angular2/src/core/testability/testability';
function _reflector() {
    return reflector;
}
/**
 * A default set of providers which should be included in any Angular platform.
 */
export const PLATFORM_COMMON_PROVIDERS = CONST_EXPR([new Provider(Reflector, { useFactory: _reflector, deps: [] }), TestabilityRegistry, Console]);
